libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteProgressChangeListenerTest.kt

```diff
@@ -334,9 +215,7 @@ class MapRouteProgressChangeListenerTest {
         every { routeLine.getExpressionAtOffset(any()) } returns expression
 
         progressChangeListener.onRouteProgressChanged(routeProgress)
-        coroutineRule.runBlockingTest {
-            progressChangeListener.onRouteProgressChanged(routeProgress)
-        }
+        progressChangeListener.onRouteProgressChanged(routeProgress)
```

Comment:
double call `progressChangeListener.onRouteProgressChanged(routeProgress)`. is it expected?

Reply:
yes. The method call being tested doesn't get called on the first route progress it should get called only on the second. Drawing a new route is a two step process.